src/main/java/appeng/core/api/definitions/ApiBlocks.java

```diff
@@ -159,70 +177,138 @@
-		this.iface = constructor.registerTileDefinition( new BlockInterface() );
-		this.cellWorkbench = constructor.registerTileDefinition( new BlockCellWorkbench() );
-		this.iOPort = constructor.registerTileDefinition( new BlockIOPort() );
-		this.condenser = constructor.registerTileDefinition( new BlockCondenser() );
-		this.energyAcceptor = constructor.registerTileDefinition( new BlockEnergyAcceptor() );
-		this.vibrationChamber = constructor.registerTileDefinition( new BlockVibrationChamber() );
-		this.quartzGrowthAccelerator = constructor.registerTileDefinition( new BlockQuartzGrowthAccelerator() );
-		this.energyCell = constructor.registerTileDefinition( new BlockEnergyCell() );
-		this.energyCellDense = constructor.registerTileDefinition( new BlockDenseEnergyCell() );
-		this.energyCellCreative = constructor.registerTileDefinition( new BlockCreativeEnergyCell() );
-		this.craftingUnit = constructor.registerTileDefinition( new BlockCraftingUnit( CraftingUnitType.UNIT ) );
-		this.craftingAccelerator = constructor.registerTileDefinition( new BlockCraftingUnit( CraftingUnitType.ACCELERATOR ) );
-		this.craftingStorage1k = constructor.registerTileDefinition( new BlockCraftingStorage( CraftingUnitType.STORAGE_1K ) );
-		this.craftingStorage4k = constructor.registerTileDefinition( new BlockCraftingStorage( CraftingUnitType.STORAGE_4K ) );
-		this.craftingStorage16k = constructor.registerTileDefinition( new BlockCraftingStorage( CraftingUnitType.STORAGE_16K ) );
-		this.craftingStorage64k = constructor.registerTileDefinition( new BlockCraftingStorage( CraftingUnitType.STORAGE_64K ) );
-		this.craftingMonitor = constructor.registerTileDefinition( new BlockCraftingMonitor() );
-		this.molecularAssembler = constructor.registerTileDefinition( new BlockMolecularAssembler() );
-		this.lightDetector = constructor.registerTileDefinition( new BlockLightDetector() );
-		this.paint = constructor.registerTileDefinition( new BlockPaint() );
-
-		this.skyStoneStair = this.makeStairs( constructor, this.skyStone_stone, "skystone.stone" );
-		this.skyStoneBlockStair = this.makeStairs( constructor, this.skyStone_block, "skystone.block" );
-		this.skyStoneBrickStair = this.makeStairs( constructor, this.skyStone_brick, "skystone.brick" );
-		this.skyStoneSmallBrickStair = this.makeStairs( constructor, this.skyStone_smallbrick, "skystone.brick.small" );
-		this.fluixStair = this.makeStairs( constructor, this.fluix, "fluix" );
-		this.quartzStair = this.makeStairs( constructor, this.quartz, "quartz.certus" );
-		this.chiseledQuartzStair = this.makeStairs( constructor, this.quartzChiseled, "quartz.certus.chiseled" );
-		this.quartzPillarStair = this.makeStairs( constructor, this.quartzPillar, "quartz.certus.pillar" );
+		this.quartzOre = registry.block( "quartz_ore", BlockQuartzOre::new )
+				.postInit( ( block, item ) ->
+				{
+					OreDictionary.registerOre( "oreCertusQuartz", new ItemStack( block ) );
```

Comment:
This was copied from the class, it just surfaces better when it's all in one place, hehe.